index.js

```diff
@@ -37,14 +37,17 @@ class Bundle {
   offline(dir, next) {
     debug('creating an offline bundle in %s', dir);
 
+    const platform = fs.existsSync(path.join(dir, 'index.ios.js'));
+    const entry = platform ? 'index.ios.js' : 'index.js';
```

Comment:
do you need to check for `index.native.js` also?

Reply:
Yes, you're right.